From 7c93327a711c3c5e0834acabd059ebd0fc70e379 Mon Sep 17 00:00:00 2001
From: Aleksander Morgado <aleksander@aleksander.es>
Date: Sun, 23 May 2021 21:46:34 +0200
Subject: [PATCH 31/39] base-bearer: ignore cancellation during sync()

There is no input cancellable in the method, so the GTask will never
get cancelled from the outside.
---
 src/mm-base-bearer.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/mm-base-bearer.c b/src/mm-base-bearer.c
index 69600c17..801c3ec1 100644
--- a/src/mm-base-bearer.c
+++ b/src/mm-base-bearer.c
@@ -1463,12 +1463,6 @@ interface_syncing_step (GTask *task)
     MMBaseBearer   *self;
     SyncingContext *ctx;
 
-    /* Don't run new steps if we're cancelled */
-    if (g_task_return_error_if_cancelled (task)) {
-        g_object_unref (task);
-        return;
-    }
-
     self = g_task_get_source_object (task);
     ctx = g_task_get_task_data (task);
 
-- 
2.31.1

